react/week2/todo-app/src/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {TodoList} from "./Todo";
```

Comment:
The common approach is to make a component folder and put js files for components there ie
import TodoList from '../components/TodoList';

You could import components files in App.js...